meteor/server/api/ingest/rundownInput.ts

```diff
@@ -1245,6 +1286,66 @@ function updateSegmentFromIngestData(
 	span?.end()
 	return hasChanged
 }
+function syncChangesToPartInstances(
+	cache: CacheForRundownPlaylist,
+	blueprint: ShowStyleBlueprintManifest,
+	notesContext: NotesContext,
+	playlist: RundownPlaylist,
+	rundown: Rundown,
+	parts: DBPart[],
+	segmentPieces: Piece[],
+	adlibPieces: AdLibPiece[],
+	adlibActions: AdLibAction[]
+) {
+	if (blueprint.syncIngestUpdateToPartInstance) {
+		if (playlist.active) {
+			const { currentPartInstance, nextPartInstance } = getSelectedPartInstancesFromCache(cache, playlist)
+			const instances: { existingPartInstance: PartInstance; playStatus: 'current' | 'next' }[] = []
+			if (currentPartInstance)
+				instances.push({ existingPartInstance: currentPartInstance, playStatus: 'current' })
+			if (nextPartInstance) instances.push({ existingPartInstance: nextPartInstance, playStatus: 'next' })
```

Comment:
I think that instead of 'next' this should be doing `isTooCloseToAutonext(...) ? 'current' : 'next'` so that the blueprints know to treat it as a 'current' part